### DISCUSSION TITLE
"No error" message

### DISCUSSION THREAD

**Discussion by HagarViking** [rtl-airband/RTLSDR-Airband#294] [Q&A]
What does the "No error" message when connecting mean? I mean, if it does not connect there must be some kind of error. At least a time-out?

"Could not connect to server.example.com:8000/mountpoint: No error"

The installation already works with another stream. The error comes from the additional output configured to send the same data to a different server. Pinging the server address is successful.

**Comment by szpajder** [comment]
> What does the "No error" message when connecting mean?
>
> "Could not connect to server.example.com:8000/mountpoint: No error"
>

It means that error messages produced by libshout library are misleading
and that it was a mistake on my part to include them in error messages
printed by rtl_airband.


> The installation already works with another stream.
>

On the same server? If yes, then you are probably hitting the max-sources
limit which is 2 by default. Increase the limit in the config of your
Icecast server.
